worldengine/generation.py

```diff
@@ -200,7 +200,7 @@ def generate_world(w, step):
 
     # Prepare sufficient seeds for the different steps of the generation
     rng = numpy.random.RandomState(w.seed)  # create a fresh RNG in case the global RNG is compromised (i.e. has been queried an indefinite amount of times before generate_world() was called)
-    sub_seeds = rng.randint(0, 4294967295, size=100)  # sys.maxsize didn't quite work
+    sub_seeds = rng.randint(0, 2147483647, size=100)  # sys.maxsize didn't quite work
```

Comment:
@psi29a Shouldn't we still just use int32? It would be a negligible loss for a medium-sized gain (less different seeds <-> reproducible worlds for everyone; especially the developers probably won't use a release version). Since you found a way to properly pull the correct number from somewhere, all would be fine in my opinion.